src/scripts/boot.lua

```diff
@@ -550,8 +550,8 @@ function love.init()
 			usedpiscale = c.window.usedpiscale,
 			x = c.window.x,
 			y = c.window.y,
+			title = c.window.title or "Untitled"
 		}), "Could not set window mode")
-		love.window.setTitle(c.window.title or c.title)
```

Comment:
Could this line be moved above the setMode call, instead of having a bunch of changes to setMode's code?

Reply:
Ah yeah, of course!

Much cleaner solution.